white-paper.md

```diff
@@ -90,35 +90,35 @@ tea’s mission is to empower open-source communities and ensure their contribut
 
 Package maintainers must make sure their software continues to deliver increasing value as the industry evolves.
 
-tea assumes that package creators maintain their work. Package maintainers are pillars of open-source communities who need to be empowered and rewarded for their ongoing contributions. A package maintainer may decide to discontinue their maintenance efforts or realize they cannot operate at a pace that matches the package users' expectations. Package maintainers receive an NFT when they complete a package submission (see the maintainer NFT section for additional details). This NFT is used to evidence their work and is the key that directs tea rewards. The holder of a package’s NFT can transfer its ownership to another developer (or group of developers), thus making them maintainers of the package and recipients of any future rewards. Similarly, a developer may decide to take on the role of package maintainer by forking the existing package and submitting a new one which they will maintain moving forward, thus becoming themselves both package creator and maintainer.
+tea assumes that package creators maintain their work. Package maintainers are pillars of open-source communities who need to be empowered and rewarded for their ongoing contributions. A package maintainer may decide to discontinue their maintenance efforts or realize they cannot operate at a pace that matches the package users' expectations. Package maintainers receive an NFT when they complete a package submission (See the [Maintainer NFT](#maintainer-nft) section for additional details). This NFT is used to evidence their work and is the key that directs tea rewards. The holder of a package’s NFT can transfer its ownership to another developer or group of developers, thus making them maintainers of the package and recipients of any future rewards. Similarly, a developer may decide to take on the role of package maintainer by forking the existing package and submitting a new one which they will maintain moving forward, thus becoming themselves both package creator and maintainer.
```

Comment:
Added a link to the "Maintainer NFT" section. Not sure if this is desired or not. I did the same everywhere else a different section is cited.

Reply:
Thanks. We had links in the document originally, but the color change (the work started on a gdoc) was rather distracting, which is why we removed the links. Here, it should not be the case, so inserting them is a good idea.
Capitalizing "See" is fine as well.